### ISSUE TITLE
[feat] sepolia deployment 

### ISSUE THREAD

**Issue by glihm** [keep-starknet-strange/piltover#27]
The idea of this PR is to implement the tooling around piltover to easily deploy the contract on Sepolia.

This includes:
1. Compiling the contracts
2. Deploying the contracts configured to interact with the herodotus verifier, [list of deployments is here](https://github.com/HerodotusDev/cairo-verifier/blob/main/deployed_contracts.md).
3. Documenting this process to be easily repeatable by user.

To achieve this task, several options:
1. Bash script with Starkli
2. Starknet foundry [scripting](https://foundry-rs.github.io/starknet-foundry/starknet/script.html)
3. Rust program using starknet-rs

We could give a try to the option `2` even if it still experimental and the API may change in the future. @b-j-roberts @drspacemn @EvolveArt what do you think it's better?
If we go with `3`, it could be a dedicated command line tool for piltover. But more stuff to maintain.

**Comment by drspacemn** [comment]
I would also vote for option `2` here, and I would prefer not to implement `3`.

**Comment by drspacemn** [comment]
@EvolveArt this is just what was merged on blobstream-starknet what are your thoughts on this vs what is done in zaun?

**Comment by thomas192** [comment]
Hey, I can work on this after it's been discussed

**Comment by glihm** [comment]
@thomas192 still around and interested to do so? Let's try with `3` rust code if you feel confortable with it.

**Comment by byteZorvin** [comment]
Hey if this is open to be taken up, I would like to work on this

**Comment by glihm** [comment]
@byteZorvin yup! Which direction would you want to take for the deployment Strat? Adding this to cainome could make the deployment process very easy from rust. :)

**Comment by byteZorvin** [comment]
Yes makes sense, even I lean towards starknet.rs scripts. If this sounds good can proceed @glihm

**Comment by glihm** [comment]
Sounds great @byteZorvin! You can go with `starknet.rs` script for now if you prefer and we can refactor later once cainome supports declare/deploy. 👍 
Or happy to have PRs on cainome if you feel about it!

**Comment by mimisavage** [comment]
Is it okay if I tackle this?

**Comment by Annyrich2** [comment]
Please I need sepolia eth 300 to 500 to test a smart contract please 


My address in address thanks please 

0x96070538C13533Cb838B2fb1799A3e5f621155eD
